buildstockbatch/cloud/docker_base.py

```diff
@@ -461,3 +465,53 @@ def run_simulations(cls, cfg, job_id, jobs_d, sim_dir, fs, output_path):
                 shutil.rmtree(item)
             elif os.path.isfile(item):
                 os.remove(item)
+
+    def log_summary(self):
+        """
+        Log a summary of how many simulations succeeded, failed, or ended with other statuses.
+        Uses the `completed_status` column of the files in results_csvs/results_*.csv.gz.
+        """
```

Comment:
I like the premise of this. When does it get called? Also, is there any way this could be in base.py so it could be available to the other batch classes?

Reply:
In the GCP branch, we're calling it after post-processing finishes ([here](https://github.com/rewiringamerica/buildstockbatch/blob/65c87039594c4d878d8996518d0de07f433a4c60/buildstockbatch/gcp/gcp.py#L917)). (So this happens on your local machine, not on GCP.)

Moving it up to the parent class should work if the subclasses all add `get_fs()` to return a file system class that can read from `self.results_dir`. That would be easy for local and AWS, but I'm not sure about the others. What do you think?